.devcontainer/Dockerfile

```diff
@@ -0,0 +1,35 @@
+# LICENSE UPL 1.0
+#
+# Copyright (c) 2021, 2022 Oracle and/or its affiliates. All rights reserved.
+#
+
+# FROM ghcr.io/graalvm/graalvm-ce:latest
+FROM container-registry.oracle.com/os/oraclelinux:7-slim
+
+RUN yum update -y oraclelinux-release-el7 \
+    && yum install -y oraclelinux-developer-release-el7 oracle-softwarecollection-release-el7 \
+    && yum-config-manager --enable ol7_developer \
+    && yum-config-manager --enable ol7_developer_EPEL \
+    && yum-config-manager --enable ol7_optional_latest \
+    && yum install -y bzip2-devel ed gcc gcc-c++ gcc-gfortran gzip file fontconfig less libcurl-devel make openssl openssl-devel readline-devel tar vi which xz-devel zlib-devel \
+    && yum install -y glibc-static libcxx libcxx-devel libstdc++-static zlib-static git \
+    && rm -rf /var/cache/yum
```

Comment:
just curious: why use `rm -rf /var/cache/yum` instead of `yum clean all`?

Reply:
I'm guessing it's not clearing everything in `/var/cache/yum` (at least, that seemed to be an [issue in OL6](https://support.oracle.com/knowledge/Oracle%20Linux%20and%20Virtualization/2157871_1.html)).

Reply:
> just curious: why use `rm -rf /var/cache/yum` instead of `yum clean all`?


Just refer to the [code here](https://github.com/graalvm/container/blob/880e5c6912796a6fd7cca1001feb66bc9a6a151f/community/Dockerfile.ol7-java17#L24).

Maybe `yum clean all` is better.

Reply:
I see thanks, let's not worry about it in this PR i was just curious.
Maybe @mzachh remembers the original reason?

it seems that regarding this OL/REHL 5/6 limitation, one alternative is to use `yum clean all --enablerepo="*"` to ensure that even disabled repos get cleaned.